pcsx2/gui/Panels/MemoryCardListView.cpp

```diff
@@ -98,9 +98,9 @@ const ListViewColumnInfo& MemoryCardListView_Simple::GetDefaultColumnInfo( uint
 	{
 		{ _("PS2 Port")		, 170 , wxLIST_FORMAT_LEFT	},
 		//{ _("Port status")	, 80  , wxLIST_FORMAT_LEFT	},
-		{ _("Card (file) name")	, 145 , wxLIST_FORMAT_LEFT	},
+		{ _("Active Memory Card")	, 145 , wxLIST_FORMAT_LEFT	},
```

Comment:
Is this not the card names in the list? they aren't *all* active, surely?

Reply:
Actually they are (slot1 and slot2). Under that are the unused cards. So it fits.

Reply:
yes but the column is saying what is underneath it, including the inactive/unmounted  cards, it's not what's active.

Reply:
Haven't thought about that way. Unless the line is replaced with a column to separate them so i can keep this. If its not feasible i'll revert the changes.

Reply:
I don't know if that is feasible, the boxes would have to be split in order to do that, but then a heading for the table would make more sense in that scenario than changing the column name

Reply:
![image](https://user-images.githubusercontent.com/24227051/91636751-57f99f80-ea03-11ea-93aa-db71e7fa3d13.png)

This is why i thought it was separated in the first place, still trying to figure out a good alternative.

Reply:
Changed Active Memory card to Memory card and Unused is changed to disabled. I tried to find my way around WX, PX and whatever is relevant to it.
Maybe unused as inactive if disabled isnt fitting?

![image](https://user-images.githubusercontent.com/24227051/91648580-8e6a0580-ea69-11ea-8ef5-d4efba2c363b.png)